alternator/stats.cc

```diff
@@ -154,6 +154,8 @@ static void register_metrics_with_optional_table(seastar::metrics::metric_groups
                     [&stats]{ return estimated_histogram_to_metrics(stats.api_operations.batch_get_item_histogram);})(op("BatchGetItem")).aggregate({seastar::metrics::shard_label}).set_skip_when_empty(),
             seastar::metrics::make_histogram("batch_item_count_histogram", seastar::metrics::description("Histogram of the number of items in a batch request"), labels,
                     [&stats]{ return estimated_histogram_to_metrics(stats.api_operations.batch_write_item_histogram);})(op("BatchWriteItem")).aggregate({seastar::metrics::shard_label}).set_skip_when_empty(),
+            seastar::metrics::make_histogram("get_item_op_size_kib", seastar::metrics::description("Histogram of item sizes involved in a request. If alternator_force_read_before_write is disabled, the metrics may be incomplete"), labels,
```

Comment:
remove the alternator_force_read_before_write part, it's more confusing than helping